workflows/metatdenovo.nf

```diff
@@ -559,6 +559,12 @@ workflow METATDENOVO {
     //
     ch_eukulele_db = Channel.empty()
     if( ! params.skip_eukulele ) {
+        // Make sure the eukulele_dbpath exists
```

Comment:
I thought we had this feature for eukulele. we probably lost during the several updates.